testsuite/domain/src/test/java/org/jboss/as/test/integration/domain/suites/IgnoredResourcesProfileCloneTestCase.java

```diff
@@ -92,14 +86,11 @@ public static void beforeClass() throws Exception {
         testSupport = DomainTestSuite.createSupport(IgnoredResourcesProfileCloneTestCase.class.getSimpleName());
         masterLifecycleUtil = testSupport.getDomainMasterLifecycleUtil();
         slaveLifecycleUtil = testSupport.getDomainSlaveLifecycleUtil();
-
-        File masterDir = new File(testSupport.getDomainMasterConfiguration().getDomainDirectory());
-        domainCfg = new File(masterDir, "configuration"
-                + File.separator + "testing-domain-standard.xml");
     }
 
     @AfterClass
     public static void afterClass() throws Exception {
+        slaveLifecycleUtil.awaitServers(System.currentTimeMillis());
```

Comment:
This isn't reliable because it only waits for servers configured with auto-start=true. It's basically written for use during a normal HC boot.

I had to work around this problem in HcExtensionAndSubsystemManagementTestCase. Kind of a pain.